assets/src/components/selectControl/index.js

```diff
@@ -83,6 +83,7 @@ const SelectControl = withFocusOutside(
 
 			return (
 				<BaseComponent
+					value={ value }
```

Comment:
Since `value` was extracted from props above, it wasn't getting passed to BaseComponent, meaning you couldn't properly set a value on selects. I've not investigated whether the same issue happens for `disabled`, but it likely has the same problem. cc @iuravic